### ISSUE TITLE
Allow USWDS Compile to have option of Sass or Sass embedded

### ISSUE THREAD

**Issue by brunerae** [uswds/uswds-compile#32]
Relates to: https://github.com/uswds/uswds-compile/issues/20 and https://github.com/uswds/uswds-compile/issues/28

Create a new setting that lets you compile Sass with plain Sass or Sass embedded.

**Comment by thisisdano** [comment]
@mejiaj I'm disinclined to pursue this solution, as it seems like an unnecessary complexity, but I'd like your opinion on the severity of the problem this would solve and the complexity of the solution.

**Comment by smustgrave** [comment]
ran into this error again for ddev same issue.

**Comment by smustgrave** [comment]
Updating sass-embeded fixed it https://github.com/uswds/uswds-compile/pull/47/files

**Comment by mejiaj** [comment]
@thisisdano agreed. Sass embedded is meant to support legacy and new API. This doesn't seem like an issue any more.

**Comment by ryparker** [comment]
Closing the alternative-compiler-setting proposal based on the maintainer decision in this discussion: the additional setting was considered unnecessary, and the original environment issue was reported resolved with an updated sass-embedded dependency. This does not assert that every compiler or platform issue is resolved.
